src/main/java/frc/robot/commands/EEBarf.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.Constants;
+import frc.robot.subsystems.EndEffector;
+
+public class EEBarf extends Command {
+  EndEffector s_EndEffector;
+  WaitCommand clock = new WaitCommand(Constants.EndEffector.waitTime);
```

Comment:
Something to keep in mind, the barf might have a different time to wait than the shooter.